library/std/src/lib.rs

```diff
@@ -545,11 +550,6 @@ pub mod arch {
 #[stable(feature = "simd_x86", since = "1.27.0")]
 pub use std_detect::is_x86_feature_detected;
 
-// The runtime entry point and a few unstable public functions used by the
-// compiler
-#[macro_use]
-pub mod rt;
-
```

Comment:
Why did this get reordered?

Reply:
The `rtabort` macro needs to be defined before the `threads` module.